driver.go

```diff
@@ -28,10 +31,11 @@ const (
 )
 
 var (
-	defaultTritonAccount = ""
-	defaultTritonKeyPath = "" // os.Getenv("HOME") + "/.ssh/id_rsa"
-	defaultTritonKeyId   = ""
-	defaultTritonUrl     = "https://us-east-1.api.joyent.com"
+	defaultTritonAccount     = ""
+	defaultTritonKeyPath     = "" // os.Getenv("HOME") + "/.ssh/id_rsa"
+	defaultTritonKeyId       = ""
+	defaultTritonKeyMaterial = ""
+	defaultTritonUrl         = "https://us-east-1.api.joyent.com"
```

Comment:
Given that we're over at MNX now, it may well be useful to just make `defaultTritonUrl` an empty string as well.